ajax/libs/meshki/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "meshki",
+  "filename": "meshki.min.js",
```

Comment:
hmmmm ... seems my last download or extraction had the problem, looks good now, thanks.